src/components/NavigationBar.jsx

```diff
@@ -9,11 +9,23 @@ class NavigationBar extends Component {
     return (
       <Row className="NavigationBar">
         <Col md={6} sm={7} xs={12} className="navigation-bar-left">
-            <IndexLink to="/" className="navbar-link" activeClassName="active">Home</IndexLink>
+            <NavLink
+              to="/"
+              className="navbar-link"
+              activeClassName="active"
```

Comment:
can you add `exact` after this line? Without exact, the `Home` link will be active for all routes (I assume cause it's the base path). With exact it'll only be highlighted for the root route and all searches